incubator/kolide-fleet/values.yaml

```diff
@@ -0,0 +1,76 @@
+---
+## Change the name of the resource
+## Default: use the chart name
+# nameOverride:
+
+mysql:
+  mysqlUser: "kolide"
+  mysqlDatabase: "kolide"
+  ## (Required)
+  mysqlPassword: ""
+
+redis:
+  ## (Required)
+  redisPassword: ""
+
+## Fleet application specific settings
+fleet:
+  image:
+    repository: "kolide/fleet"
+    tag: "latest"
+    pullPolicy: "Always"
+  replicas: 1
+  pod:
```

Comment:
I understand your concerns but it is commonly done as follows. Please change accordingly.

```yaml 
fleet:
  podAnnotations:
  resources: {}
```